docs/content/menu/index.md

```diff
@@ -3,74 +3,27 @@ headless: true
     - [Build properties]({{< ref "/docs/projects/BuildProperties.md" >}})
     - [Build trace]({{< ref "/docs/projects/BuildTrace.md" >}})
     - [CI Steps]({{< ref "/docs/projects/CiSteps.md" >}})
 - Contributing
-    - [How to start]({{< ref "/docs/contributing/HowToStart.md" >}})
-    - [Documentation]({{< ref "/docs/contributing/Docs.md" >}})
-    - [Code Review]({{< ref "/docs/contributing/CodeReview.md" >}})
-    - [Code Style]({{< ref "/docs/contributing/CodeStyle.md" >}})
-    - [Feature Toggles]({{< ref "/docs/contributing/FeatureToggles.md" >}})
-- IDE
-    - [Конфигурации]({{< ref "/docs/ide/Configurations.md" >}})
-    - [Шаблоны]({{< ref "/docs/ide/Templates.md" >}})
-    - [Speedup]({{< ref "/docs/ide/Speedup.md" >}})
-    - [Бенчмарки]({{< ref "/docs/ide/Benchmarking.md" >}})
-- Builds
-    - [Типы сборки]({{< ref "/docs/assemble/BuildTypes.md" >}})
-    - [Dependencies]({{< ref "/docs/assemble/Dependencies.md" >}})
-    - [Минимизация]({{< ref "/docs/assemble/Minimization.md" >}})
-    - [Бенчмарки]({{< ref "/docs/assemble/Benchmarks.md" >}})
-    - [Профилирование]({{< ref "/docs/assemble/Profiling.md" >}})
-    - [Метрики]({{< ref "/docs/assemble/BuildMetrics.md" >}})
-    - [Удаленная сборка (Mirakle)]({{< ref "/docs/assemble/RemoteBuild.md" >}})
-    - [Кеширование]({{< ref "/docs/assemble/BuildCaching.md" >}})
-    - [Troubleshooting]({{< ref "/docs/assemble/Troubleshooting.md" >}})
-- Architecture
-    - [Dagger]({{< ref "/docs/architecture/Dagger.md" >}})
-    - [Modules]({{< ref "/docs/architecture/Modules.md" >}})
-- [Аналитика]({{< ref "/docs/analytics/Analytics.md" >}})
-    - [Statsd]({{< ref "/docs/analytics/Statsd.md" >}})
+    - [Documentation]({{< ref "/docs/contributing/documentation.md" >}})
 - Testing
-    - [Запуск тестов]({{< ref "/docs/test/Run.md" >}})
     - [Test Runner]({{< ref "/docs/test/Runner.md" >}})
-    - [Ручное]({{< ref "/docs/test/Manual.md" >}})
-    - [Test case in code]({{< ref "/docs/test/TestCaseInCode.md" >}})
-    - [Юнит тесты]({{< ref "/docs/test/UnitTesting.md" >}})
-    - [Скриншотные тесты]({{< ref "/docs/test/ScreenshotTesting.md" >}})
-    - [Test minimized]({{< ref "/docs/test/TestMinimized.md" >}})
-    - [Report Viewer]({{< ref "/docs/test/ReportViewer.md" >}})
-    - [Troubleshooting]({{< ref "/docs/test/TroubleshootingUI.md" >}})
-    - [Work Manager Testing]({{< ref "/docs/test/WorkManager.md" >}})
     - [Flaky annotation]({{< ref "/docs/test/FlakyAnnotation.md" >}})
-    - [Нестабильные тесты]({{< ref "docs/test/FlakyTests.md" >}})
     - [Toast]({{< ref "/docs/test/Toast.md" >}})
-    - [Snackbar]({{< ref "/docs/test/Snackbar.md" >}})
-    - [Screen checks]({{< ref "/docs/test/ScreenChecks.md" >}})
 - [Test framework]({{< ref "/docs/test_framework/TestFramework.md" >}})
-    - [Mocking]({{< ref "/docs/test_framework/Mocking.md" >}})
+    - [Snackbar]({{< ref "/docs/test_framework/Snackbar.md" >}})
+    - [Screen checks]({{< ref "/docs/test_framework/ScreenChecks.md" >}})
```

Comment:
Decided to keep pages about the test framework and libraries here.
I want to have them in the repository with the code. Will remove them from internal documentation.